### PR TITLE
Name from content (2ee8b8), add substitute math symbols failure

### LINKED ISSUE #493: Issue with rule "Label and name from content mismatch" (2ee8b8)
@mraccess77 wrote in https://github.com/act-rules/act-rules.github.io/issues/104#issuecomment-488006872: 

> I think "2" and "two" would pass as they would be voiced the same way be users of speech recognition and this is the reason for this test. Some discussion here w3c/wcag#592.

### PR DESCRIPTION
**Pull request by WilcoFiers**
Name from content, add substitute math symbols failure

Closes #493

## Guidance for the PR (pull request) creator

**When creating PR:**
- [ ] Make sure you requesting to **pull a issue/feature/bugfix branch** (right side) to the master branch (left side).

**After creating PR:**
- [ ] Add yourself (and co-authors) as "Assignees" for PR
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore` (more to the administrative side)
- [ ] Add relevant project (e.g. "Q3 2018 Status") to PR
- [ ] OPTIONAL: If you want anyone in particular to review your pull request, assign them as "Reviewers".
- [ ] Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)

# How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.

### CODE REVIEW

File: _rules/label-content-name-mismatch-2ee8b8.md [hunk 1]
```diff
@@ -93,6 +93,14 @@ Not all of visible label is included in accessible name.
 <button name="link" aria-label="the full">The full label</button>
 ```
 
+#### Failed Example 3
+
+Mathematical symbols can not be substitute for text.
+
+```html
+<a href="/" aria-label="Page two">Page 2</a>
```
Comment (jkodu) [inline _rules/label-content-name-mismatch-2ee8b8.md]: can we add examples with other mathematical operators (+,-,*,/).

Comment (WilcoFiers) [reply _rules/label-content-name-mismatch-2ee8b8.md]: In the spirit of not overloading the test cases, I'd prefer not to. Anyone else things this is essential? @kasperisager @carlosapaduarte @annethyme?

Comment (kasperisager) [reply _rules/label-content-name-mismatch-2ee8b8.md]: I do think calling numerals "mathematical symbols" is a bit of a stretch.

Comment (carlosapaduarte) [reply _rules/label-content-name-mismatch-2ee8b8.md]: Indeed, I have to agree with @kasperisager 
That would be appropriate if the example included a mathematical formula, but here the issue is not related with the interpretation of mathematical symbols, but with potential translation issues.
My suggestion would be to retain this example but with a different description, and add an example with a mathematical formula with this example's description.

Comment (ShadowBB) [reply _rules/label-content-name-mismatch-2ee8b8.md]: Can we have one example that includes both?

`<a href="/" aria-label="Proof of one plus one is two">Proof of 1+1=2</a>`

**Comment by jkodu** [comment]
Linked PR - https://github.com/act-rules/act-rules.github.io/pull/452

We have to ensure that there are no merge conflicts and or no difference in interpretation of the rule.

**Comment by jkodu** [comment]
@WilcoFiers can you resolve conflicts here and get this going again please.

**Comment by Jym77** [comment]
This is minor change and will only require a 1 week Final Call.

File: _rules/visible-label-in-accessible-name-2ee8b8.md [hunk 2]
```diff
@@ -93,6 +93,14 @@ Not all of [visible][] label is included in [accessible name][].
 <button name="link" aria-label="the full">The full label</button>
 ```
 
+#### Failed Example 3
+
+Mathematical symbol cannot be substitute for text as [explicitly mentioned in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name#mathematical-expressions-and-formulae).
```
Comment (Brynanders) [inline _rules/visible-label-in-accessible-name-2ee8b8.md]: Mathematical symbol cannot be substitute... > Mathematical symbols cannot be substituted...

Comment (audreymaniez) [reply _rules/visible-label-in-accessible-name-2ee8b8.md]: +1 @Brynanders "Mathematical symbols cannot be substituted by text"

Comment (Jym77) [reply _rules/visible-label-in-accessible-name-2ee8b8.md]: Can actually go both ways, depending on which one is the "original" and which one is the [substitute](https://en.wiktionary.org/wiki/substitute#Noun) 😄

Comment (Brynanders) [reply _rules/visible-label-in-accessible-name-2ee8b8.md]: @Jym77 its grammatically incorrect to use the singular "symbol" and "substitute"
symbol > symbols
substitute > substituted

Comment (Jym77) [reply _rules/visible-label-in-accessible-name-2ee8b8.md]: Ah, yes, I missed the requested pluralisation of "symbol" 😞 Makes sense…

**Comment by Jym77** [comment]
Final Call ends on October 29.

**Comment by bruce-usab** [comment]
I like adding the math example, but I think the aria label should be “Proof of two times two equals four” and not “Proof of two multiplied by two is four”.  (A more literal reading might help with translation too.)

File: _rules/visible-label-in-accessible-name-2ee8b8.md [hunk 3]
```diff
@@ -93,6 +93,14 @@ Not all of [visible][] label is included in [accessible name][].
 <button name="link" aria-label="the full">The full label</button>
 ```
 
+#### Failed Example 3
+
+Mathematical symbols cannot be substituted for text as [explicitly mentioned in WCAG](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name#mathematical-expressions-and-formulae).
+
+```html
+<a href="/" aria-label="Proof of two multiplied by two is four">Proof of 2&times;2=4</a>
```
Comment (bruce-usab) [inline _rules/visible-label-in-accessible-name-2ee8b8.md]: I like adding the math example, but aria-label should be “Proof of two times two equals four” and not “Proof of two multiplied by two is four”, as per the guidance in the Understanding page link.   (Also, a more literal aria-label might help with translation too.)

**Comment by Jym77** [comment]
> I like adding the math example, but I think the aria label should be “Proof of two times two equals four” and not “Proof of two multiplied by two is four”. (A more literal reading might help with translation too.)

That's a bit of the point 😃 The example is failing because math symbols cannot be used on one side (accessible name/label) while text is used for the other. WCAG lists as a reason the fact that `×` can be read both at "times" and "multiplied by":
>  For example, making the name "eleven multiplied by three is equivalent to thirty-three" might mean a user who said "eleven times three equals thirty-three" may not match.

Hence, I voluntary chose "multiplied by" (which I consider as being a less common way to pronounce `×`, and visually clashing with `&times;`) in order to illustrate why this is a problem (in addition to linking to the longer explanation in WCAG).

**Comment by bruce-usab** [comment]
Yikes!  I totally missed that this was a failure example!  Thank you @Jym77!

**Pull request merged**